stl/inc/xutility

```diff
@@ -5563,17 +5563,42 @@ _NODISCARD constexpr auto lexicographical_compare_three_way(
 }
 #endif // __cpp_lib_concepts
 
+template <class _Ty, class _Elem>
+struct _Vector_alg_in_find_is_safe_object_pointers : false_type {};
+template <class _Ty1, class _Ty2>
+struct _Vector_alg_in_find_is_safe_object_pointers<_Ty1*, _Ty2*>
+    : conjunction<
+          // _Ty1* is an object pointer type

```

Comment:
I don't know if there's a better way to implement this. LWG give us `is_object_pointer_v` you cowards!

Reply:
The only other way I know how to express this is "not a function type", but I think that's less clear. (It would also be horrible for throughput if we still relied on the library-only implementation.)